src/components/TaskCard/AddTaskCard.jsx

```diff
@@ -35,6 +35,10 @@ const AddTaskCard = ({ isCardCreated, onCreateCard }) => {
     }
   }
 
+  function handleDeleteTask() {
+    onCreateCard(!isCardCreated);
```

Comment:
probable misuse of prop, consider introducing `onDeleteTask` prop to propagate state of task deleting.